src/m365/spo/commands/homesite/homesite-list.ts

```diff
@@ -5,20 +5,26 @@ import SpoCommand from '../../../base/SpoCommand.js';
 import commands from '../../commands.js';
 import { CliRequestOptions } from "../../../../request.js";
 
-class SpoTenantHomeSiteListCommand extends SpoCommand {
+class SpoHomeSiteListCommand extends SpoCommand {
   public get name(): string {
-    return commands.TENANT_HOMESITE_LIST;
+    return commands.HOMESITE_LIST;
   }
 
   public get description(): string {
     return 'Lists all home sites';
   }
 
+  public alias(): string[] | undefined {
+    return ['spo tenant homesite list'];
+  }
+
   public defaultProperties(): string[] | undefined {
     return ['Url', 'Title'];
   }
 
   public async commandAction(logger: Logger): Promise<void> {
+    await this.showDeprecationWarning(logger, this.alias()![0], commands.HOMESITE_LIST);
```

Comment:
I think I'd prefer keeping TENANT_HOMESITE_LIST as a constant in options for now, so we can reference it in this file in several places, including this one instead of this.alias().... just like we do it in other places.

Would you agree?